bigchaindb/commands/utils.py

```diff
@@ -130,6 +158,10 @@ def mongodb_host(host):
                          help='Specify the location of the configuration file '
                               '(use "-" for stdout)')
 
+base_parser.add_argument('-l', '--log-level',
+                         choices=['DEBUG', 'INFO', 'WARNING', 'ERROR', 'CRITICAL'],
```

Comment:
The new `-l` / `--log-level` CLI options (and the allowed choices of level) should be documented in the server docs, in https://docs.bigchaindb.com/projects/server/en/master/server-reference/bigchaindb-cli.html